### DISCUSSION TITLE
Mapping Next.js .env.development to NOW environment variables

### DISCUSSION THREAD

**Discussion by inian** [vercel/vercel#4360] [General] [answered]
I am using .env.development, .env.production files in Next.js 9.4

How do I achieve the following setup? 
I use Now for my Next.js application. I want to test my PRs to master and all my branches to use my development env variables. Since `npm run build` uses production mode always which picks up the env variables from .env.production. For testing branches and PRs, I do not want the production env variables to be used. 

Or is there a way for Now to pick up the environment variables set in .env.development file? I do not want to manually set the env variables in Now since there is duplication that I want to avoid.

**Comment by paulogdm** [comment, marked as answer]
Hello,

You need to use our [environments variable UI](https://vercel.com/blog/environment-variables-ui) to add the required variables. Can you give it a try and let us know?
